apps/mark-scan/frontend/src/pages/ballot_invalidated_page.test.tsx

```diff
@@ -29,6 +32,7 @@ function renderWithAuthAndBallotContext(
 ): VxRenderResult {
   return render(
     <BallotInvalidatedPage
+      logger={logger}
```

Comment:
Feels like we need a `useLogger` hook.

Reply:
In line with discussion in https://github.com/votingworks/vxsuite/pull/4592#discussion_r1488741427, [579f510](https://github.com/votingworks/vxsuite/pull/4569/commits/579f510b785d3fe76d0162cdfec727c36210606b) moves logging to the backend